aleph_alpha_client/embedding.py

```diff
@@ -56,3 +64,62 @@ def from_json(json: Dict[str, Any]) -> "EmbeddingResponse":
             },
             tokens=json.get("tokens"),
         )
+
+
+class SemanticRepresentation(Enum):
+    """
+    Available types of semantic representations that prompts can be embedded with.
+    """
+
+    Symmetric = "symmetric"
+    """
```

Comment:
? I do not think there is pydoc for members, is there? Usually this is just added to the doc of the class.